ep/build-velox/src/build_velox.sh

```diff
@@ -117,6 +127,9 @@ function compile {
     echo "ENABLE_BENCHMARK is ON. Disabling Tests, GCS and ABFS connectors if enabled."
     COMPILE_OPTION="$COMPILE_OPTION -DVELOX_ENABLE_BENCHMARKS=ON"
   fi
+  if [ $ENABLE_VCPKG == "ON" ]; then
```

Comment:
Seems we can just rely on this check to see whether the below env variable is set:
`if [ -n "${GLUTEN_VCPKG_ENABLED:-}" ];`